plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/StmtConversionContext.kt

```diff
@@ -238,6 +239,7 @@ fun StmtConversionContext.convertMethodWithBody(
     // note: we must guarantee somewhere that returned value is Unit
     // as we may not encounter any `return` statement in the body
     returnTarget.variable.withIsUnitInvariantIfUnit().toViperUnusedResult(linearizer)
+    body.checkValidity(declaration, errorCollector)
```

Comment:
Doesn't need to be in this pull request, but I assume we want to stop things if there was an error.